library/std/src/sys/windows/os.rs

```diff
@@ -318,13 +318,33 @@ pub fn temp_dir() -> PathBuf {
     super::fill_utf16_buf(|buf, sz| unsafe { c::GetTempPath2W(sz, buf) }, super::os2path).unwrap()
 }
 
-#[cfg(not(target_vendor = "uwp"))]
+#[cfg(and(not(target_vendor = "uwp"), not(target_vendor = "win7")))]
+fn home_dir_crt() -> Option<PathBuf> {
+    unsafe {
+        // Defined in processthreadsapi.h.
+        const CURRENT_PROCESS_TOKEN: usize = -4isize as usize;
```

Comment:
On a stylistic note, could you also add an underscore to visually separate the numeric value from the type? This is just to help people who are scanning the code. I.e.:

```suggestion
        const CURRENT_PROCESS_TOKEN: usize = -4_isize as usize;
```